andela/riders-ah-frontend
# #165413081 list bookmarked articles

Description
=======
This feature is about allowing an authenticated user to see articles they bookmarked in their profile.

Type of change
=======
N/A

How has it been tested
=======
- pull this branch
- run ```npm install```
- run ```npm start``` to start our react app
-login into authorshaven
-open your profile.
-if you have bookmarked articles you will see them. if not you will see a message indicating you have none.
-Click on the title or image of the bookmarked article to navigate to it.

Checklist:
=======
N/A

Pivotal tracker story ID
=======
[#165413081](https://www.pivotaltracker.com/story/show/165413081)

Screenshot
=======
<img width="1080" alt="Screen Shot 2019-07-29 at 12 50 53" src="https://user-images.githubusercontent.com/43787407/62214275-05cff000-b3a5-11e9-9931-e17ac79ac573.png">
